### ISSUE TITLE
SteamOS UI doesn't start anymore [Segmentation fault]

### ISSUE THREAD

**Issue by sir-ragna** [ValveSoftware/SteamOS#583]
#### System information

I can't sign into the F7 TTY anymore to verify these.
This is the version information of the last time I checked.

- Steam client version: 1482202200 or higher.
- SteamOS version: 2.0 1:2.98 or higher
- Opted into Steam client beta?: No
- Opted into SteamOS beta?: No

#### Please describe your issue in as much detail as possible:

- When I boot up I get a black screen. The default F7 TTY doesn't start the SteamOS UI anymore. It is possible however to sign into the desktop on the F2 TTY.
- From the logs I can see there was a segmentation fault. I am appending the dump and log files.
[steamdump.zip](https://github.com/ValveSoftware/SteamOS/files/755771/steamdump.zip)
[dmesg.txt](https://github.com/ValveSoftware/SteamOS/files/755772/dmesg.txt)

#### Steps for reproducing this issue:

I can reproduce this by rebooting my system.

The last system modification I did (outside of installing games on SteamOS) was installing Perl6 Rakudo from the apt-get package manager.

#### Next steps

- I will see what happens after uninstalling Perl6(built on the Parrot engine).
- Update with the package manager
- Gather more dumps.

**Comment by kisak-valve** [comment]
Minor note, the attached steamdump.zip is empty.

**Comment by sir-ragna** [comment]
@kisak-valve sorry let me try that again.

[steamdump (1).zip](https://github.com/ValveSoftware/SteamOS/files/755851/steamdump.1.zip)

**Comment by sir-ragna** [comment]
Meanwhile I removed Perl6 (the rakudo debian package) from the system.
No package updates are available. (I commented out the only non-default provider in the sources.list file.)

The problem persists however.

See attachment for `dpkg -l` output. I think kept my installed packages to an absolute minimum.

[dpkg-l.txt](https://github.com/ValveSoftware/SteamOS/files/755905/dpkg-l.txt)

I can provide more dumps/logs/config files if requested.
